besu/src/main/java/org/hyperledger/besu/cli/options/TransactionPoolOptions.java

```diff
@@ -277,6 +279,25 @@ static class Unstable {
         arity = "1")
     private Duration eth65TrxAnnouncedBufferingPeriod =
         TransactionPoolConfiguration.Unstable.ETH65_TRX_ANNOUNCED_BUFFERING_PERIOD;
+
+    @CommandLine.Option(
+        names = {MAX_TRACKED_SEEN_TXS_PER_PEER},
+        paramLabel = "<LONG>",
+        hidden = true,
+        description =
+            "The number of exchanged txs that are remembered with each peer, to avoid broadcasting duplicates (default: ${DEFAULT-VALUE})",
+        arity = "1")
+    private int maxTrackedSeenTxsPerPeer =
+        TransactionPoolConfiguration.Unstable.DEFAULT_MAX_TRACKED_SEEN_TXS_PER_PEER;
+
+    @CommandLine.Option(
+        names = {PEER_TRACKER_FORGET_EVICTED_TXS_FLAG},
+        paramLabel = "<BOOLEAN>",
+        hidden = true,
```

Comment:
Does this need a fallback value so `--Xpeer-tracker-forget-evicted-txs` will work without "=true"?

Reply:
Good point, the important thing is that _fallback value_ does not ct as a default, because in this case the default is chosen at runtime depending on the txpool implementation